app/src/main/java/com/keylesspalace/tusky/components/viewthread/edits/ViewEditsAdapter.kt

```diff
@@ -0,0 +1,190 @@
+
+        val edit = edits[position]
+
+        val binding = holder.binding
+
+        val context = binding.root.context
+
+        val avatarRadius: Int = context.resources
+            .getDimensionPixelSize(R.dimen.avatar_radius_48dp)
+
+        loadAvatar(edit.account.avatar, binding.statusEditAvatar, avatarRadius, animateAvatar)
+
+        val infoStringRes = if (position == edits.size - 1) {
+            R.string.status_created_info
+        } else {
+            R.string.status_edit_info
+        }
+
+        val timestamp = absoluteTimeFormatter.format(edit.createdAt, false)
+
+        binding.statusEditInfo.text = context.getString(
+            infoStringRes,
+            edit.account.name,
+            timestamp
+        ).emojify(edit.account.emojis, binding.statusEditInfo, animateEmoji)
+
+        if (edit.spoilerText.isEmpty()) {
+            binding.statusEditContentWarningDescription.hide()
+            binding.statusEditContentWarningSeparator.hide()
+        } else {
+            binding.statusEditContentWarningDescription.show()
+            binding.statusEditContentWarningSeparator.show()
+            binding.statusEditContentWarningDescription.text = edit.spoilerText.emojify(
+                edit.emojis,
+                binding.statusEditContentWarningDescription,
+                animateEmoji
+            )
+        }
+
+        val emojifiedText = edit.content.parseAsMastodonHtml().emojify(edit.emojis, binding.statusEditContent, animateEmoji)
+        setClickableText(binding.statusEditContent, emojifiedText, emptyList(), emptyList(), listener)
+
+        if (edit.poll == null) {
+            binding.statusEditPollOptions.hide()
+            binding.statusEditPollDescription.hide()
+        } else {
+            binding.statusEditPollOptions.show()
+
+            // not used for now since not reported by the api
+            // https://github.com/mastodon/mastodon/issues/22571
+            // binding.statusEditPollDescription.show()
+
+            val pollAdapter = PollAdapter()
+            binding.statusEditPollOptions.adapter = pollAdapter
+            binding.statusEditPollOptions.layoutManager = LinearLayoutManager(context)
+
+            pollAdapter.setup(
+                options = edit.poll.options.map { it.toViewData(false) },
+                voteCount = 0,
+                votersCount = null,
+                emojis = edit.emojis,
+                mode = if (edit.poll.multiple) { // not reported by the api
+                    MULTIPLE
+                } else {
+                    SINGLE
+                },
+                resultClickListener = null,
+                animateEmojis = animateEmoji,
+                enabled = false
+            )
+
+           /*binding.statusEditContentWarningDescription.text = context.getString(
+                R.string.poll_info_time_absolute,
+                absoluteTimeFormatter.format(edit.poll.expiresAt, false)
+            ) */
```

Comment:
```suggestion
```

?

Reply:
Yeah that is the info I'd like to set but it we don't get it from the api.